### ISSUE TITLE
Offset buttons in HTML

### ISSUE THREAD

**Issue by kgaeb** [DataCoveEU/API4INSPIRE#97]
when displaying items in the HTML implement buttons for previous/next and a label showing the displayed numbers from/to of the features

**Comment by kgaeb** [comment]
could you please add to "Displaying items: 560 - 570" the total number of items
eg.:
Displaying items: 560 - 570 (out of 576)

**Comment by TobiasPressler** [comment]
> could you please add to "Displaying items: 560 - 570" the total number of items
> eg.:
> Displaying items: 560 - 570 (out of 576)

As far as I know there it is not possible to get the total amount of items from an OGC API. It would only be possible by checking all items and therefor spamming the API routes.

**Comment by LukasGaebler** [comment]
OGC API Specification requirement 30 & 31:
http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_response_6

It says that the server **if possible and not difficult to compute**  may compute the number of features that match the request.
As Tobias already wrote, it would only be possible by spamming the API routes. We would call the API route for the items and count how many items there are. Then we would call the "next" link, to get the the next items. We would then add the number of items we got to the number we have counted before.
